src/com/google/javascript/jscomp/CodeGenerator.java

```diff
@@ -1085,7 +1085,7 @@ protected void add(Node n, Context context) {
             checkState(
                 !isInClass, "initializers should only exist in object literals, not classes");
             cc.addOp(":", false);
```

Comment:
hmmm...
Given my statement on the next line to make this consistent with `addStringKey()`, I think this line should probably be
`cc.add(":")`

It's not technically correct to consider this colon an operator. I'm not sure it does any harm as it is currently written, but it does invoke a bunch of unnecessary logic.